src/orga/guidelines.rst

```diff
@@ -164,3 +164,21 @@ The guidance that we can provide here is two fold:
    For example, we're holding off on turning off py36 until pypy comes out with pypy3.7.
 3. If there are lots of people in the community relying on older versions, core team can decide to keep an old version around
    For example, we held off turning off py27 even after numpy, scipy dropped support as there were many in the community interested in keeping support until the end of life of that version.
+
+
+Requesting ``core`` to Merge PRs
+================================
+
+Due to various reasons, it is sometimes necessary for the ``core`` team to merge PRs on feedstocks they do not personally
+maintain. The ``core`` team abides by the following guidelines when doing this.
+
+1. The ``core`` team will normally wait three days before merging a PR on a feedstock they do not maintain personally. This
```

Comment:
This should be a week.

Reply:
We discussed at the core meeting and everyone seemed to think three days is fine. I am personally not willing to put a week in this rule, as it is too long. By the same token, a day is too short. We are trying to find a middle ground here.

Reply:
Three days is too short to reasonably expect feedback from volunteers who are putting in their own free time. To me, this feels like core is trying to take over maintenance of all feedstocks, rather than being the maintainer of last resort.